### ISSUE TITLE
`atom` command not opening current folder

### ISSUE THREAD

**Issue by sorccu** [atom/atom#7245]
I had originally installed atom via `brew cask install` and the `atom` command was working fine. Then, for reasons, I uninstalled the cask version and installed atom from atom.io. Ever since, despite performing "Install Shell Commands" and making sure the symlinks are correct, the `atom` command fails to load the current folder. It's highly likely this is isolated to my installation only, but I'm having a hard time figuring out where things go wrong.

I have tried:

* Removing ~/.atom entirely
* Exporting a nearly uncustomized `$PATH` before launching atom
* Atom -> Install Shell Commands (and manually checking the symlinks after)
* Launching from various folders, just in case there was something weird in the pathname

So far nothing has worked.

Atom seems to be getting the `--executed-from` argument just fine and with a correct value. For example, if I launch atom in /tmp, I can see the following command with `ps aux | grep -i atom`:

```
/Applications/Atom.app/Contents/MacOS/Atom --executed-from=/tmp --pid=77617 --path-environment=/Users/redacted/.bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin
```

The launched editor just shows a blank untitled document, does not open the tree view, and Cmd+P says that the project is empty. The `--pid` doesn't point to any existing process but then again I don't really know what it's for anyway.

File -> Open does work.

Any ideas? I'm on 0.209.0 currently but the issue has persisted for a month or so.

Here's the path to the atom script in case it helps:

```
➜  /tmp  readlink /usr/local/bin/atom
/Applications/Atom.app/Contents/Resources/app/atom.sh
```

**Comment by mnquintana** [comment]
That is intended behavior – see https://github.com/atom/atom/issues/6464 or https://github.com/atom/atom/pull/6690 for more context. To open the current directory in `atom`, just run `atom .` like other command line programs.

Duplicate of https://github.com/atom/atom/issues/6865

**Comment by sorccu** [comment]
Thanks, I had not noticed that the behavior changed. Looking closer now, this change is indeed included in the release notes for v0.199.0. So I was simply blind, thanks again!
